### ISSUE TITLE
How would I take an image from Cloudinary, to Netlify CMS, into a gatsby-image element?

### ISSUE THREAD

**Issue by gusbmurphy** [gatsbyjs/gatsby#23466]
Hi! Thanks in advance for your attention to this novice question. 

## Summary
I'm fairly new to Gatsby and web development as a whole, and probably just need to talk through a problem I'm having trouble with. What I'm trying to accomplish is to make use of `gatsby-image` with images that are stored in Cloudinary, that are referenced in markdown files created by Netlify CMS.

I'm using `gatsby-image` in order to take advantage of the lazy loading capabilities, Netlify CMS because I'm interested in using it to make a blog, portfolio, or storefront page, and then Cloudinary because it's presented itself as a solution to Netlify CMS's problem with selecting multiple images in a single image field (my conclusion I've reached with this is that it requires a third party media library).

## My understanding of the problem
To restate, I am a novice with this whole world of things, but I'm going to try my best to describe my understanding of this. In order to use `gatsby-image` I need to be working with ImageSharp nodes, however because my images are being hosted on Cloudinary, what I end up querying in my JavaScript is simply a bunch of URL's to those images. 

I think that because I'm kind of putting myself on a crash course, I feel like I may have chosen tools that are not really well-suited to the thing I'm trying to accomplish: allowing a user to upload images in a CMS, and then select multiple images in a single field on something like a blog entry, and ultimately have those images be rendered with lazy-loading with `gatsby-image`. Have I introduced too much complexity with both Netlify CMS and Cloudinary? Is there some other service I should be using for that? My apologies again if this is a too broad of a question, but again I feel that at my stage in learning about all of this I'd really appreciate simply talking through some stuff!

**Comment by jbFeDev** [comment]
@gusbmurphy I have the same question, and I think it's a good question. I was able to use ImageSharp and gatsby-image within my project without any problem, but when I switched to Cloudinary media library, I also get plain absolute urls to the images (which are fun). But now we should be able to process them the same way as gatsby processes relative images in markdown. I also hope there will be an answer somehow soon :)

**Comment by ku8ar** [comment]
@gusbmurphy hi! Did you found any solution?

**Comment by Kcharle** [comment]
I would also love to know if there is a way to get this working.

**Comment by gusbmurphy** [comment]
@ku8ar , @Kcharle sorry but I've been away from this project for a while! I'm going to be revisiting it soon though so I may return to this issue.

**Comment by casperleerink** [comment]
Also would like to know more about this!

**Comment by lemurweb-dev** [comment]
I'd too be interested in a solution / workaround for this!

**Comment by bytrangle** [comment]
Not sure about others' setup, but mine is here
[Repo](https://github.com/bytrangle/gatsby-netlify-cms)
[Codesandbox](https://codesandbox.io/s/gatsby-netlify-cms-cloudinary-59nlx?file=/src/pages/index.js)

I configure Netlify CMS to use Cloudinary for media mangagement. This means that when I create a new post in Netlify CMS and insert images, Netlify CMS will create a Markdown file and insert the Cloudinary url like this:

```
![acrylic-paint](https://res.cloudinary.com/bytrangle/image/upload/v1628569271/Code%20demos/pawel-czerwinski-NSg56vIPotw-unsplash_emthmu.jpg "Blue acrylic paint")
```
When this Markdown file is turned into a HTML page, the `<img` tag will have absolute, remote url in its `src` attribute.

```
<p><img src="https://res.cloudinary.com/bytrangle/image/upload/v1628569271/Code%20demos/pawel-czerwinski-NSg56vIPotw-unsplash_emthmu.jpg" alt="acrylic-paint" title="Blue acrylic paint"></p>
```
Here is what the page looks like on the deployed site:

![markdown-page-img-tag](https://user-images.githubusercontent.com/28924121/128995045-bbe4c598-ca5d-4b37-baea-02a834fac447.png)

I can't use `gatsby-remark-images` for processing the image to make it responsive or avoid layout jumps because `gatsby-remark-images` only works with relative image path.

In other words, AFAIK there's no way to use `gatsby-image`, `gatsby-plugin-image`, gatsby-remark-images` to process images in Markdown file that are remote Urls. 

Would love to hear other people's approach.
